src/TestcafeGherkinBootstrapper.js

```diff
@@ -140,20 +140,28 @@ module.exports = class TestcafeGherkinBootstrapper extends TestcafeBootstrapper
   }
 
   _shouldRunScenario(scenario) {
-    if (!this.tags.length) {
-      return true;
-    }
+    return this._scenarioHasAnyOfTheTags(scenario, this._getIncludingTags(this.tags)) &&
```

Comment:
This line does not comply with this project's code style. Please run prettier (`npm run prettier`) before committing changes to fix this automatically. :)
You can also use a file watcher for your IDE (see [Prettier docs](https://prettier.io/docs/en/editors.html)) to format your code automatically.